app/views/comments/_submit_comment_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= form_with model: comment, data: {turbo: false}  do |f| %>
+    <%= render 'shared/error_messages', object: f.object %>
+    <%= f.hidden_field :post_id, value: post.id %>
+    <%= f.hidden_field :user_id, value: current_user.id %>
```

Comment:
czy tak to na pewno powinno być? A co jeśli ktoś zły podmieni user_id na user_id innego użytkownika? Wtedy bedzie mogl dodawac komenatrze jako ktoś inny.

Czy tu controller nie ma jakoś opcji authoryzować użytkownika i z autoryzacji brać user-id?

Reply:
Poprawione.